db/schema.rb

```diff
@@ -43,7 +43,7 @@
   end
 
   create_table "bookings", force: :cascade do |t|
-    t.boolean "confirmed", default: false
+    t.boolean "confirmed"
```

Comment:
Was a migration file created for this change? Because when i run migration it reverts it to default:false locally

Reply:
Did we get rid of default false so we could have some venues booked and some not when seeding?